dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -5159,7 +5159,9 @@ public void updateLogOffset(String logFile, Long newLogOffset) {
                             String regexp = it.next();
                             if (Pattern.compile(regexp).matcher(line).find()) {
                                 it.remove();
-                                break;
+                                //There used to be a break here but if a user passed in a
```

Comment:
I feel guilty walking past the egregious pattern recompile for every line. I shudder to think how many times the smarter planet regex is compiled for each build.

Reply:
That's a good point, caching even just the basic smarter planet regex would probably be a good improvement